homeassistant/components/buienradar/camera.py

```diff
@@ -44,6 +44,10 @@ async def async_setup_entry(
     latitude = config.get(CONF_LATITUDE, hass.config.latitude)
     longitude = config.get(CONF_LONGITUDE, hass.config.longitude)
 
+    if None in (latitude, longitude):
+        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
+        return
+
```

Comment:
lat and lon seems mandatory in the config flow so this guard should not be needed + change above to only get from entry options.